### PR TITLE
[Improve][Connectors]update httpclient version to 4.5.13 because of doris connection reset

### PR DESCRIPTION
**Pull request by zhaomin1423**
## Purpose of this pull request
update httpclient version to 4.5.13 because of doris connection reset.
refer doc: https://doris.apache.org/zh-CN/administrator-guide/load-data/stream-load-manual.html#%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98
The FAQ3 is account for the pr.

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in you PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/developement/NewLicenseGuide.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by CalvinKirs** [comment]
hi,please refer this :https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/developement/NewLicenseGuide.md

resolve license error.

**Comment by zhaomin1423** [comment]
> hi,please refer this :https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/developement/NewLicenseGuide.md
> 
> resolve license error.

Thanks.

**Comment by zhaomin1423** [comment]
> Some checks were not successful



> hi,please refer this :https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/developement/NewLicenseGuide.md
> 
> resolve license error.

Hi, Some checks were not successful. what do I need to do?

**Comment by CalvinKirs** [comment]
> > Some checks were not successful
> 
> > hi,please refer this :https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/developement/NewLicenseGuide.md
> > resolve license error.
> 
> Hi, Some checks were not successful. what do I need to do?

and need add this :  `commons-codec-1.11.jar`
```
+commons-codec-1.11.jar
```

**Comment by zhaomin1423** [comment]
> commons-codec-1.11.jar

Thanks

File: seatunnel-dist/release-docs/LICENSE [hunk 1]
```diff
@@ -329,7 +329,7 @@ The text of each license is the standard Apache 2.0 license.
      (Apache License, Version 2.0) Apache Hadoop YARN Client (org.apache.hadoop:hadoop-yarn-client:3.0.0 - no url defined)
      (Apache License, Version 2.0) Apache Hadoop YARN Common (org.apache.hadoop:hadoop-yarn-common:3.0.0 - no url defined)
      (Apache License, Version 2.0) Apache HttpAsyncClient (org.apache.httpcomponents:httpasyncclient:4.1.2 - http://hc.apache.org/httpcomponents-asyncclient)
-     (Apache License, Version 2.0) Apache HttpClient (org.apache.httpcomponents:httpclient:4.5.2 - http://hc.apache.org/httpcomponents-client)
+     (Apache License, Version 2.0) Apache HttpClient (org.apache.httpcomponents:httpclient:4.5.13 - http://hc.apache.org/httpcomponents-client)
      (Apache License, Version 2.0) Apache HttpClient Mime (org.apache.httpcomponents:httpmime:4.5.2 - http://hc.apache.org/httpcomponents-client)
```
Comment (CalvinKirs) [inline seatunnel-dist/release-docs/LICENSE]: you need to add `commons-codec` here,

I checked NOTICE and he didn't change, so just make the changes here.

Comment (zhaomin1423) [reply seatunnel-dist/release-docs/LICENSE]: This process is tedious, can we simplify it？

Comment (CalvinKirs) [reply seatunnel-dist/release-docs/LICENSE]: License is very important, it is required. I don't currently see a better approach (automation) for other projects.

Comment (zhaomin1423) [reply seatunnel-dist/release-docs/LICENSE]: Before add common-codec, the license checker is successful. now, it is failed.

Comment (CalvinKirs) [reply seatunnel-dist/release-docs/LICENSE]: That's not your problem, hi, @asdf2014 can you take a look at this?

Comment (zhaomin1423) [reply seatunnel-dist/release-docs/LICENSE]: > That's not your problem, hi, @asdf2014 can you take a look at this?

Thanks.

Comment (asdf2014) [reply seatunnel-dist/release-docs/LICENSE]: @zhaomin1423 Have you looked the log

Comment (asdf2014) [reply seatunnel-dist/release-docs/LICENSE]: @CalvinKirs It is normal, we should be glad that the checker is working

Comment (CalvinKirs) [reply seatunnel-dist/release-docs/LICENSE]: Am I missing something? I saw that the license item has been done, and this error is very strange.

File: seatunnel-dist/release-docs/LICENSE [hunk 2]
```diff
@@ -329,7 +329,8 @@ The text of each license is the standard Apache 2.0 license.
      (Apache License, Version 2.0) Apache Hadoop YARN Client (org.apache.hadoop:hadoop-yarn-client:3.0.0 - no url defined)
      (Apache License, Version 2.0) Apache Hadoop YARN Common (org.apache.hadoop:hadoop-yarn-common:3.0.0 - no url defined)
      (Apache License, Version 2.0) Apache HttpAsyncClient (org.apache.httpcomponents:httpasyncclient:4.1.2 - http://hc.apache.org/httpcomponents-asyncclient)
-     (Apache License, Version 2.0) Apache HttpClient (org.apache.httpcomponents:httpclient:4.5.2 - http://hc.apache.org/httpcomponents-client)
+     (Apache License, Version 2.0) Apache HttpClient (org.apache.httpcomponents:httpclient:4.5.13 - http://hc.apache.org/httpcomponents-client)
+     (Apache License, Version 2.0) Apache Commons Codec (commons-codec:commons-codec:1.11 - https://commons.apache.org/proper/commons-codec)
```
Comment (asdf2014) [inline seatunnel-dist/release-docs/LICENSE:333]: @zhaomin1423 You shouldn’t add this line, please delete this line, which is displayed at log as well. Cc @CalvinKirs

File: seatunnel-dist/release-docs/LICENSE (hunk 1 above)
Comment (asdf2014) [reply seatunnel-dist/release-docs/LICENSE]: @CalvinKirs see my last comment

File: seatunnel-dist/release-docs/LICENSE (hunk 2 above)
Comment (CalvinKirs) [reply seatunnel-dist/release-docs/LICENSE:333]: I think this is a bug of Python check, this jar should be added. (if you look at his dependencies).

**Review by CalvinKirs — APPROVED** [review]
LGTM

File: seatunnel-dist/release-docs/LICENSE (hunk 2 above)
Comment (asdf2014) [reply seatunnel-dist/release-docs/LICENSE:333]: @CalvinKirs Emm.. I'm afraid this is not about the Python script itself, the reason was that the `license-maven-plugin` maven plugin is not stable when the sortArtifactByName option is enabled, and I will raise up another PR to enhance the plugin.

Comment (CalvinKirs) [reply seatunnel-dist/release-docs/LICENSE:333]: OK, I will merge this PR and look forward to your refinement.

Comment (asdf2014) [reply seatunnel-dist/release-docs/LICENSE:333]: @CalvinKirs Agree, thanks

Comment (asdf2014) [reply seatunnel-dist/release-docs/LICENSE:333]: It has been solved by https://github.com/apache/incubator-seatunnel/pull/1387

**Pull request merged**